src/main/kotlin/no/nav/familie/ef/sak/service/VedtaksbrevService.kt

```diff
@@ -0,0 +1,43 @@
+package no.nav.familie.ef.sak.service
+
+import no.nav.familie.ef.sak.api.dto.BrevRequest
+import no.nav.familie.ef.sak.brev.BrevClient
+import no.nav.familie.ef.sak.repository.VedtaksbrevRepository
+import no.nav.familie.ef.sak.repository.domain.Vedtaksbrev
+import no.nav.familie.ef.sak.service.steg.StegType
+import org.springframework.stereotype.Service
+import java.util.*
+
+@Service
+class VedtaksbrevService(private val brevClient: BrevClient,
+                         private val brevRepository: VedtaksbrevRepository,
+                         private val behandlingService: BehandlingService,
+                         private val fagsakService: FagsakService,
+                         private val personService: PersonService) {
+
+    fun lagBrev(behandlingId: UUID): ByteArray {
+        val behandling = behandlingService.hentBehandling(behandlingId)
+        val fagsak = fagsakService.hentFagsak(behandling.fagsakId)
+        val person = personService.hentSøker(fagsak.hentAktivIdent())
+        val navn = person.navn.get(0)
```

Comment:
viktig å huske; data fra pdl kommer som lister og vi har implementert en funksjon `gjeldende` for alle disse listene:

`val navn = person.navn.gjeldende().visningsnavn()`  vil gi riktig fornavn, mellomnavn og etternavn.

Reply:
Ja, det virket litt "hacky" å skrive `navn.get(0)` 🙈